rules/call_peak.smk

```diff
@@ -1,24 +1,33 @@
+def get_genrich_replicates(wildcards):
+    assembly_ish, sample_condition = "-".join(wildcards.fname.split('-')[:-1]), wildcards.fname.split('-')[-1]
+    assembly = assembly_ish.split("/")[-1]
+
+    if sample_condition in samples.index or (sample_condition in treps.index and config.get("technical_replicates", "") == "keep"):
+        print(sample_condition, "here")
+        return expand(f"{{dedup_dir}}/{wildcards.fname}.sambamba-queryname.bam", **config)
+    else:
+        return expand([f"{{dedup_dir}}/{assembly}-{replicate}.sambamba-queryname.bam"
+        for replicate in samples[(samples['assembly'] == assembly) & (samples['condition'] == sample_condition)].index], **config)
```

Comment:
should this not be
`for replicate in treps[(treps['assembly'] == assembly) & (treps['condition'] == sample_condition)].index], **config)`
to get the replicate names (if they exist)?

Also, this whole section assumes `condition` is in samples.

Reply:
I think I actually should use `treps_from_brep` here.